build.gradle

```diff
@@ -346,6 +347,7 @@ subprojects { subproject ->
     implementation group: 'org.apache.cxf', name: 'cxf-rt-transports-http', version: cxfVersion
 
     implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: log4jVersion
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: log4jVersion
```

Comment:
Just wondering why log4j core is needed?

Reply:
The log4j core dependency is needed because the performance logger is a log4j logger (see https://github.com/hmcts/civil-sdt/blob/master/utils/src/main/java/uk/gov/moj/sdt/utils/logging/PerformanceLogger.java#L133).  Without log4j-core the PerformanceLoggerTest tests fail because it can't locate the classes it needs.

Reply:
Ok. We'll probably resolve this later as I'd think the alternatives will be in the newer jars.